bridge/engine/AudioForwarderReceiveJob.cpp

```diff
@@ -147,6 +158,7 @@ void AudioForwarderReceiveJob::run()
         }
 
         _activeMediaList.onNewAudioLevel(_sender->getEndpointIdHash(), audioLevel);
+        _activeMediaList.onNewPtt(_sender->getEndpointIdHash(), is_ptt);
 
         if (audioLevel >= _silenceThresholdLevel)
```

Comment:
Is this really correct? The silenceThreshold is 127. Iirc the audioLevel in packet has omitted sign, so level is 1 - pktLevel.
So >= is correct to detect it is now completely silent. But the default value if there is no audio level in packet is -1. Why not 0 then.
I think even the ext.data[0] is unsigned.